<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeScreen</class>
 <widget class="QDialog" name="HomeScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>272</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>480</width>
    <height>272</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HomeScreen</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(30, 30, 30, 255);</string>
  </property>
  <widget class="QPushButton" name="infoButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(200, 200, 150, 255);
background: qradialgradient(cx:.4, cy:.2, radius:1, fx:.5, fy:.3, stop:.2 rgba(70, 70, 70, 255), stop:.9 rgba(20, 20, 20, 255));
border-style: solid;
border-width: 3px;
border-radius: 30px;
border-color: rgba(115, 115, 95, 255);
font: 18px &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>System Info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pktsButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>0</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(200, 200, 150, 255);
background: qradialgradient(cx:.4, cy:.2, radius:1, fx:.5, fy:.3, stop:.2 rgba(70, 70, 70, 255), stop:.9 rgba(20, 20, 20, 255));
border-style: solid;
border-width: 3px;
border-radius: 30px;
border-color: rgba(115, 115, 95, 255);
font: 18px &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>UL Packets</string>
   </property>
  </widget>
  <widget class="QPushButton" name="settingsButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>160</width>
     <height>60</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(200, 200, 150, 255);
background: qradialgradient(cx:.4, cy:.2, radius:1, fx:.5, fy:.3, stop:.2 rgba(70, 70, 70, 255), stop:.9 rgba(20, 20, 20, 255));
border-style: solid;
border-width: 3px;
border-radius: 30px;
border-color: rgba(115, 115, 95, 255);
font: 18px &quot;DejaVu Sans&quot;;</string>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>421</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="dateLabel">
      <property name="styleSheet">
       <string notr="true">color: rgba(255, 255, 255, 255);
font: 18px &quot;DejaVu Sans&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="timeLabel">
      <property name="styleSheet">
       <string notr="true">color: rgba(255, 255, 255, 255);
font: 18px &quot;DejaVu Sans&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="uptimeLabel">
      <property name="styleSheet">
       <string notr="true">color: rgba(255, 255, 255, 255);
font: 18px &quot;DejaVu Sans&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ipLabel">
      <property name="styleSheet">
       <string notr="true">color: rgba(255, 255, 255, 255);
font: 18px &quot;DejaVu Sans&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="gpsLabel">
      <property name="styleSheet">
       <string notr="true">color: rgba(255, 255, 255, 255);
font: 18px &quot;DejaVu Sans&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/rsrc/logo.png</pixmap>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>infoButton</zorder>
  <zorder>pktsButton</zorder>
  <zorder>settingsButton</zorder>
  <zorder>label</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>infoButton</sender>
   <signal>clicked()</signal>
   <receiver>HomeScreen</receiver>
   <slot>slot1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pktsButton</sender>
   <signal>clicked()</signal>
   <receiver>HomeScreen</receiver>
   <slot>slot2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>239</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>settingsButton</sender>
   <signal>clicked()</signal>
   <receiver>HomeScreen</receiver>
   <slot>slot3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot1()</slot>
  <slot>slot2()</slot>
  <slot>slot3()</slot>
 </slots>
</ui>
